packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/cta.tsx

```diff
@@ -139,7 +140,16 @@ const CTA: React.FC<Props> = ({
         </Text>
       ) : null}
       {disabled ? null : (
-        <div style={{ display: 'flex' }}>
+        <div
```

Comment:
Same here, also many styles are being duplicated, you could define an styled component in a `.model` file for being reused